src/crud.js

```diff
@@ -0,0 +1,44 @@
+const localTaskKey = 'task.list';
+const localTasks = JSON.parse(localStorage.getItem(localTaskKey)) || [];
+const taskInput = document.querySelector('[data-task-input]');
+
+const clearElement = (element) => {
+  while (element.firstChild) {
+    element.removeChild(element.firstChild);
+  }
+};
+
+const createTask = () => ({
+  id: Date.now().toString(),
+  description: taskInput.value,
+  completed: false,
+});
+
+const save = () => {
+  localStorage.setItem(localTaskKey, JSON.stringify(localTasks));
+};
+
+const editTask = (e) => {
+  const edit = e.target;
+  if (edit.tagName.toLowerCase() === 'textarea') {
+    // eslint-disable-next-line eqeqeq
```

Comment:
- [OPTIONAL] It would be better to not disable linter rules in your files. It will help you to make clean and readable code. ♻️